frontend/components/channel-sidebar/channel_list.jsx

```diff
@@ -6,17 +6,10 @@ class ChannelList extends React.Component{
 
   render(){
     const channels = Object.values(this.props.channels).filter(
-      convo =>  !convo.is_dm
+      convo => !convo.is_dm
     );
     const channelEls = channels.map( channel => {
-      if (!channel.private || this.props.permissions[channel.id]){
-        return <ChannelListItem
-                  key={channel.id}
-                  channel={channel}
-                 />;
-      } else {
-        return '';
-      }
+        return <ChannelListItem key={channel.id} channel={channel} />;
```

Comment:
in this render you have two iterations, one uses implicit return, and the other explicit. consistency. might be even cleaner if you extract all that logic and iteration into a separate method